draft-ietf-rtcweb-security-arch.xml

```diff
@@ -995,6 +996,9 @@ Syntax:
           Implementations MUST NOT implement DTLS renegotiation and MUST reject
           it with a "no_renegotiation" alert if offered.</t>
 
+        <t>
+          Implementations MUST NOT implement TLS False Start <xref target="RFC7918"/>.</t>
```

Comment:
I think that this is "endpoint MUST NOT enable false start".  Because we're definitely not going to be compliant based on this text.

Reply:
Funny I also wrote that because it's really just about the client.